tests/unit/models/v0.3/modelCard.test.js

```diff
@@ -155,7 +157,7 @@ describe("CardValidator", () => {
       card.address.validate = oldValidate;
     });
 
-    it("should update the errors object in the card if any errors are returned", async () => {
+    it.skip("should update the errors object in the card if any errors are returned", async () => {
```

Comment:
This is a separate issue but not necessarily a bug. If Service Objects throws an error or says an address is invalid for the `/validations/address` endpoint, we return that message. For the `/patrons` endpoint, however, if there's an error validating an address, we want to continue anyway and do a naive check on the city and state fields. This will return a temporary card to the user but it won't return an error. This is business logic planned out since Service Objects can be down for whatever reason.

https://jira.nypl.org/browse/DQ-415